template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/raz/RangerRazBaseConfigProvider.java

```diff
@@ -30,7 +30,7 @@
             Set<HostgroupView> hostgroupViews = source.getHostgroupViews();
 
             return hostgroupViews.stream()
-                    .filter(hg -> hg.getName().toLowerCase().equals("master"))
+                    .filter(hg -> hg.getName().toLowerCase().equals("master") || hg.getName().toLowerCase().equals("main"))
```

Comment:
Hmm.. Didn't think about this.. Did you run a search-all on all code for the string `"master"`. Hopefully this doesn't break things arbitrarily.

Reply:
It deploys fine.  But I'll look further into if anything uses that name.

Reply:
There's 3196 occurrences of "master" in CB.

Reply:
> There's 3196 occurrences of "master" in CB.
:/

I am wondering if we should split this into two tasks
1. Changing the shape
2. Changing the name "master" to "main"

That way, if something does cause an issue with hardcoded string "master", we can look into that separately and we would not need to rollback the whole shape change if it comes down to that. What do you think?

Reply:
I'm going to do the rename of master->main in another commit so if we have to change it back, we can just revert that commit instead of the whole commit.  There's a bunch of user text and integration tests that use the name master.  Maybe we should just name gateway master (since it runs CM anyway).